packages/nova-routing/lib/server/routing.jsx

```diff
@@ -40,6 +40,7 @@ Meteor.startup(() => {
     },
     wrapperHook(req, res, appGenerator) {
       const { apolloClient, store } = getRenderContext();
+      store.reload({ message: 'reload store before render' });
```

Comment:
Yes - we have action creators/reducers/middleware in our implementation that I would like to include in the SSR mechanism.